src/apify_client/clients/resource_clients/dataset.py

```diff
@@ -271,7 +271,6 @@ def download_items(
         # We need to override and then restore the warnings filter so that the warning gets printed out,
         # Otherwise it would be silently swallowed
         with warnings.catch_warnings():
-            warnings.simplefilter('always')
```

Comment:
Now the `warnings.catch_warnings():` context manager is unnecessary, because we're not changing the warnings filter anymore, we could remove this whole logic and just keep the `warnings.warn(...)` call

Reply:
Sure, makes sense, I removed the related comment lines as well.